src/main/java/uk/gov/hmcts/reform/sendletter/entity/StaleLetter.java

```diff
@@ -3,49 +3,15 @@
 import java.util.Date;
 import java.util.UUID;
 
-public class StaleLetter {
+public interface StaleLetter {
```

Comment:
Since we are using projection I would recommend you rename this class as `StaleLettersOnly` as it would be very clear then rather than sounding like a POJO.